### ISSUE TITLE
[Feature request] don't turn off power if printer is printing

### ISSUE THREAD

**Issue by calania** [jneilliii/OctoPrint-Domoticz#6]
Hello, I have been using this plugin for a while and it is working perfectly. I just have one problem. I have added the M81 command so that it turns off the printer 300 seconds after the print is complete. This normally works well except when I want to start a new print right after the other finished. Now I need to wait for your plugin to turn off the power before starting the new print otherwise it would kill the power mid print. What I would want is an option so that the plugin only turns off the power if the printer isn't printing. Hope this can be added!

**Comment by jneilliii** [comment]
I need to check, but I think if the warn while printing option is enabled the off command won't be triggered.

**Comment by calania** [comment]
It still triggers even if that option is enabled. Just saw that there was another issue post https://github.com/jneilliii/OctoPrint-Domoticz/issues/5. It looks like you added it to a beta version but never made a stable release

**Comment by jneilliii** [comment]
Good point. Looks like that branch is 2 versions ahead of the releases. I'll try to push that official tonight.

**Comment by calania** [comment]
Sounds good!
